lib/src/lib.rs

```diff
@@ -173,9 +173,15 @@ impl<const N: usize> LoraE5<N> {
     pub fn send(&mut self, data: &[u8], port: u8, confirmed: bool) -> Result<Option<Downlink>> {
         self.set_port(port)?;
         let end_line = if confirmed {
-            "+CMSGHEX: Done\r\n"
+            "+CMSGHEX: Start\r\n"
```

Comment:
Maybe keep end_line as it was and make your "Starts" be start_line. With your logic below, you would do `read_until_pattern(&[start_line, busy_line])` and then `read_until_pattern(&[end_line])`?

Reply:
Resolved.